openrtx/src/core/graphics.c

```diff
@@ -569,6 +580,21 @@ point_t gfx_print(point_t start, fontSize_t size, textAlign_t alignment,
     return gfx_printBuffer(start, size, alignment, color, text);
 }
 
+point_t gfx_printSymbol(point_t start, fontSize_t size, textAlign_t alignment,
```

Comment:
The function definition needs to be added to graphics.h:
```
point_t gfx_printSymbol(point_t start, fontSize_t size, textAlign_t alignment,
                  color_t color, const char *fmt, ... );
```

Reply:
Doh!